backends/platform/sdl/win32/win32_wrapper.cpp

```diff
@@ -23,12 +23,19 @@
 #define FORBIDDEN_SYMBOL_EXCEPTION_strcat
 
 #define WIN32_LEAN_AND_MEAN
-#include <windows.h>
-#include <shellapi.h> // for CommandLineToArgvW()
-#if defined(__GNUC__) && defined(__MINGW32__) && !defined(__MINGW64_VERSION_MAJOR)
+#include <sdkddkver.h>
+#if !defined(_WIN32_IE) || (_WIN32_IE < 0x500)
  // required for SHGetSpecialFolderPath and SHGFP_TYPE_CURRENT in shlobj.h
+#undef _WIN32_IE
 #define _WIN32_IE 0x500
 #endif
+#if !defined(_WIN32_WINNT) || (_WIN32_WINNT < 0x500)
+ // required for VER_MAJORVERSION, VER_MINORVERSION and VER_GREATER_EQUAL in winnt.h
+#undef _WIN32_WINNT
+#define _WIN32_WINNT 0x500
```

Comment:
It would be good to add a comment here mentioning that we set this for Windows 2000

https://learn.microsoft.com/en-us/cpp/porting/modifying-winver-and-win32-winnt?view=msvc-170

Reply:
Done.